### ISSUE TITLE
TypeScript definition

### ISSUE THREAD

**Issue by zveljkovic** [aexmachina/factory-girl#138]
Any plans to create TypeScript definition files?

**Comment by juanpaucar** [comment]
@zveljkovic  There was a PR for that on definitely typed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30135

**Comment by elliotf** [comment]
I think this can be closed, since DefinitelyTyped/DefinitelyTyped#30135 was merged, right?

**Comment by dchenk** [comment]
The declarations should be part of this package.

**Comment by willtpwise** [comment]
The types in DefinitelyTyped are not complete... It's missing a lot of definitions, for example, the adaptors.

**Comment by v1b3m** [comment]
> The types in DefinitelyTyped are not complete... It's missing a lot of definitions, for example, the adaptors.

Yeah, having an issue with that `Property 'SequelizeAdapter' does not exist on type 'Static'.`
